Tools/Frame_params/Sub/bluerov2-3_5.params

```diff
@@ -333,7 +333,7 @@ NTF_LED_BRIGHT,3
 NTF_LED_OVERRIDE,0
 PILOT_ACCEL_Z,100
 PILOT_THR_FILT,0.000000000000000000
-PILOT_VELZ_MAX,500
+PILOT_VELD_MAX,500
```

Comment:
I don't think this is a correct change because this should be a parameter name.  I don't think sub has ever had a parameter named PILOT_VELD_MAX.  I think we should just remove this change because this parameter file is also version specific.

What do you think @Williangalvani?